src/MongoDB.Driver.Core/Core/NativeLibraryLoader/RelativeLibraryLocatorBase.cs

```diff
@@ -21,6 +21,9 @@ namespace MongoDB.Driver.Core.NativeLibraryLoader
 {
     internal abstract class RelativeLibraryLocatorBase : ILibraryLocator
     {
+        // public properties
+        public virtual bool IsX32ModeSupported => false;
```

Comment:
This is a temporal property to support x32 mode with snappy. This property should be overridden in `Snappy32NativeMethods.cs` and `Snappy64NativeMethods`